src/versions/validation/event.js

```diff
@@ -101,6 +123,10 @@ schemas['2019-05-20'].create = {
 
 const validationVersions = {
   '2019-05-20': [
+    {
+      target: 'event.getHistory',
+      schema: schemas['2019-05-20'].getHistory
```

Comment:
Shouldn’t you add this endpoint to future version?

Reply:
Yes right, I'll add this endpoint to a new version.